Remove unnecessary pacages

### Bug 발생 시의 상황 묘사

Build failed by type error

### 확인한 브라우저 (optional, mutiple)

_No response_

### 관련 Log 출력 첨부 (optional)

```shell
warn  - No build cache found. Please configure build caching for faster rebuilds. Read more: https://nextjs.org/docs/messages/no-cache
Attention: Next.js now collects completely anonymous telemetry regarding usage.
This information is used to shape Next.js' roadmap and prioritize features.
You can learn more, including how to opt-out if you'd not like to participate in this anonymous program, by visiting the following URL:
https://nextjs.org/telemetry

info  - Checking validity of types...
Failed to compile.

./pages/posts/[slug].tsx:69:23
Type error: Type '() => void | Transformer<Root, Root>' is not assignable to type 'Pluggable<[(Settings | undefined)?], Settings>'.
  Type '() => void | Transformer<Root, Root>' is not assignable to type 'Plugin<[(Settings | undefined)?], Settings>'.
    Type 'void | Transformer<Root, Root>' is not assignable to type 'void | Transformer'.
      Type 'Transformer<Root, Root>' is not assignable to type 'void | Transformer'.
        Type 'Transformer<Root, Root>' is not assignable to type 'Transformer'.
          Types of parameters 'node' and 'node' are incompatible.
            Property 'children' is missing in type 'Node<Data>' but required in type 'Root'.

  67 |         emoji,
  68 |       ],
> 69 |       rehypePlugins: [rehypeSlug, rehypeAutolinkHeadings],
     |                       ^
  70 |     },
  71 |     scope: data,
  72 |   });
npm ERR! code ELIFECYCLE
npm ERR! errno 1
npm ERR! react-blog@0.1.0 build: `next build`
npm ERR! Exit status 1
npm ERR! 
npm ERR! Failed at the react-blog@0.1.0 build script.
npm ERR! This is probably not a problem with npm. There is likely additional logging output above.

npm ERR! A complete log of this run can be found in:
npm ERR!     /home/runner/.npm/_logs/2022-02-08T08_00_09_563Z-debug.log
Error: Process completed with exit code 1.
```

--- a/pages/posts/[slug].tsx
+++ b/pages/posts/[slug].tsx
@@ -66,7 +66,7 @@ export const getStaticProps: GetStaticProps = async ({ params }) => {
         require("remark-sub-super"),
         emoji,
       ],
-      rehypePlugins: [rehypeSlug, rehypeAutolinkHeadings],
+      rehypePlugins: [],
     },
     scope: data,
   });

